src/vk/vk_util.c

```diff
@@ -22,16 +22,59 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 #include <assert.h>
 
+/*
+Returns number of bits set to 1 in (v).
+
+On specific platforms and compilers you can use instrinsics like:
+
+Visual Studio:
+    return __popcnt(v);
```

Comment:
I guess it should work then ?

Reply:
Should work, but i don't have VS for check.

I closed previous by mistake #25